lib/helpers/cursor/eachAsync.js

```diff
@@ -23,22 +23,20 @@ const promiseOrCallback = require('../promiseOrCallback');
 
 module.exports = function eachAsync(next, fn, options, callback) {
   const parallel = options.parallel || 1;
-  const batchSize = options.batchSize;
+  const batchSize = options.batchSize || undefined;
```

Comment:
Why is this necessary? For one thing, this will cause `batchSize: 0` to throw an unreadable error.

Reply:
Correct.